sonar-plugin-api/src/main/java/org/sonar/api/resources/Qualifiers.java

```diff
@@ -89,17 +89,24 @@ public static boolean isView(final Resource resource, final boolean acceptSubVie
   /**
    * @param resource not nullable
    */
+  public static boolean isApplication(final Resource resource) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "Resource"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)